keps/sig-node/5394-psi-node-conditions/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: PSI based Node Conditions
+kep-number: 5394
+authors:
+  - "@ndixita"
+  - "@dragoncell"
+  - "@roycaihw"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
```

Comment:
I believe you want sig-scheduling here, since you've mentioned in the document that you want to sync with them about modifying `TaintNodesByCondition`